modules/distribution/product/pom.xml

```diff
@@ -322,6 +322,18 @@
                             <goal>run</goal>
                         </goals>
                     </execution>
+                    <execution>
+                        <id>modify-scope-validators-micro-gw</id>
+                        <phase>prepare-package</phase>
+                        <configuration>
+                            <tasks>
+                                <replace file="${basedir}/../../p2-profile/micro-gateway/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/identity/identity.xml" token="&lt;ScopeValidator class=&quot;org.wso2.carbon.identity.oauth2.validators.XACMLScopeValidator&quot;/&gt;" value="&lt;!--ScopeValidator class=&quot;org.wso2.carbon.identity.oauth2.validators.XACMLScopeValidator&quot;/--&gt;" />
```

Comment:
@npamudika - Shall we try to remove this tag using the same plugin rather than commenting it? IMO it should be possible.